backend/setup.py

```diff
@@ -144,6 +144,8 @@
         ],
         "console_scripts": ["tracimcli = tracim_backend.command:main"],
         "tracimcli": [
+            # workspace
+            "space_move = tracim_backend.command.space:MoveSpaceCommand",
```

Comment:
@PhilippeAccorsi suggest that having 
"tracimcli move space" is more clear than "tracimcli space move"
the drawback is that this does not follow the current name organization where we put the element (caldav, user, db, search, webdav) before the action.

Reply:
i added this suggest to #1497 as this need a more deep rethinking of the ui.